src/helpers/verifyOtp.ts

```diff
@@ -0,0 +1,10 @@
+import OtpModel from "../model/OTP";
+
+export async function verifyOtp(email: string, otp: number) {
+  const OTPData = await OtpModel.findOne({ email });
```

Comment:
Add error handling for the findOne query in case the database fails or the connection is lost.